public/app/plugins/panel/stat/types.ts

```diff
@@ -65,6 +66,24 @@ export function addStandardDataReduceOptions(
     showIf: currentConfig => currentConfig.reduceOptions.values === false,
   });
 
+  if (includeFieldMatcher) {
+    const options = [
+      { value: '', label: 'Numeric Fields' },
+      { value: '/.*/', label: 'All Fields' },
+    ];
+    // TODO? list all fields here...
```

Comment:
@dprokop -- can the options be loaded dynamically?  based on the current query results?